### PR TITLE
VR-5865 Support options for Endpoint.update_from_config

### PR DESCRIPTION
**Pull request by nhatsmrt**
Requires resource dict-ization in https://github.com/VertaAI/modeldb/pull/1251
Please review the config file format to see if it makes sense.

### CODE REVIEW

File: client/verta/tests/test_endpoint/test_endpoint.py [hunk 1]
```diff
@@ -424,3 +425,60 @@ def test_update_autoscaling(self, client, created_endpoints, experiment_run, mod
...
+        endpoint = client.set_endpoint(path)
+        created_endpoints.append(endpoint)
+
+        original_status = endpoint.get_status()
+        original_build_ids = get_build_ids(original_status)
+
+        # Creating config dict:
+        config_dict = {
+            "run_id": experiment_run.id,
+            "strategy": "direct",
```
Comment (convoliution) [inline client/verta/tests/test_endpoint/test_endpoint.py:446]: This should _also_ work with canary and its rules, right?

**Review by convoliution — APPROVED** [review]

File: client/verta/tests/test_endpoint/test_endpoint.py (hunk 1 above)
Comment (nhatsmrt) [reply client/verta/tests/test_endpoint/test_endpoint.py:446]: No reason to believe otherwise 😬

**Comment by convoliution** [comment]
My goodness git, we're just adding tests. Why do the conflicts have to be so complex

**Comment by nhatsmrt** [comment]
> My goodness git, we're just adding tests. Why do the conflicts have to be so complex

Dw about it. Lemme handle this :(

**Comment by nhatsmrt** [comment]
Conflict resolved. And tests still pass.

**Pull request merged**